protocol-units/bridge/contracts/src/NativeBridge.sol

```diff
@@ -54,10 +59,12 @@ contract NativeBridge is AccessControlUpgradeable, PausableUpgradeable, INativeB
 
         // Set insurance fund
         insuranceFund = _insuranceFund;
-        riskDenominator = MINIMUM_RISK_DENOMINATOR + 1;
+        riskDenominator = RISK_DENOMINATOR_LOWER_BOUND + 1;
```

Comment:
```suggestion
        riskDenominator = RISK_DENOMINATOR_LOWER_BOUND;
```

Reply:
not good since the lowerbound determines the minimum excluding bound. It has to be 3 because the check has to be >, not >= (saves some computation).